parlai/tasks/md_gender/yelp.py

```diff
@@ -189,9 +189,7 @@ def _setup_data(self, opt):
         extra_data = []
         if self.add_unknown_classes:
             # load about data (unknown but inferred)
-            extra_data = gend_utils.get_inferred_about_data(
-                self.opt['task'], self.opt['datatype']
-            )
+            extra_data = gend_utils.get_inferred_about_data(self.opt['task'], self.opt)
```

Comment:
was this a bug?

Reply:
hmm I imagine that the function signature must have changed somewhere along the way?